packages/react-cards/src/components/Card/examples/Card.Compact.Example.tsx

```diff
@@ -0,0 +1,61 @@
+// @codepen
+import * as React from 'react';
+import { Text } from '@uifabric/experiments';
+import { Icon, Image, Stack } from 'office-ui-fabric-react';
+import { mergeStyleSets } from 'office-ui-fabric-react/lib/Styling';
+import { Card } from '../Card';
+
+const alertClicked = (): void => {
+  alert('Clicked');
+};
+export class CardCompactExample extends React.Component<{}, {}> {
+  public render(): JSX.Element {
+    const styles = mergeStyleSets({
+      siteText: {
+        color: '#025F52',
+        fontSize: 14,
+        fontWeight: 600
+      },
+      descriptionText: {
+        color: '#33332D',
+        fontSize: 16,
+        fontWeight: 600
+      },
+      helpfulText: {
+        color: '#323130',
+        fontSize: 14,
+        fontWeight: 600
+      }
+    });
+
+    return (
+      <Stack gap={20}>
+        <Card compact={true} gap={12} tokens={{ padding: 12 }}>
+          Basic compact card
+        </Card>
+
+        <Card compact={true} gap={12} tokens={{ padding: 12 }} onClick={alertClicked}>
+          <Card.Item tokens={{ margin: '-12px 0 -12px -12px' }}>
```

Comment:
Not for change now, but I would like to see no hardcoded gap sizes or paddings.

Allowing or encouraging hardcoded sizes like this in the code I think will have some bad consequences. People will have opinions which will create inconsistency.

It also makes the implementation messy.

Reply:
I removed both `gap` and `padding` hardcoded sizes for the `Card` usage in the examples. Sadly, I didn't remove the `margins` in `Card.Item` because it would break some inner styling right now.